entry.go

```diff
@@ -99,6 +99,11 @@ func (te *TunaEntry) Start(shouldReconnect bool) error {
 		log.Printf("Serving %s on localhost udp port %v", te.Service.Name, udpPorts)
 	}
 
+	if !te.IsServer && te.ServiceInfo.IPFilter.NeedGeoInfo() {
+		te.ServiceInfo.IPFilter.AddProvider()
```

Comment:
It's better to use
```go
go func() {
    te.ServiceInfo.IPFilter.AddProvider()
    te.ServiceInfo.IPFilter.UpdateDataFile()
}
```
so that tuna can provide service before db update finish